app/(authenticated)/(tabs)/index.tsx

```diff
@@ -7,26 +7,51 @@ import {
   Pressable,
 } from "react-native";
 import { tw } from "@/utils/tailwind";
-import { userData } from "@/constants/userData";
 import { UserInfo } from "@/utils/types/user";
 import { StatusBar } from "expo-status-bar";
+import { getFriendsList, getUserById } from "@/database/queries/user";
+import { useEffect, useState } from "react";
 
 const HomeScreen = () => {
-  const handleSelectedUser = () => {
-    // link to chat room (chatId) if there is an active room
-    // if no active room, create a new chat room and redirect to room
+  const [friendsList, setFriendsList] = useState<UserInfo[]>();
+  console.log("🚀 ~ HomeScreen ~ friendsList:", friendsList);
+  const userId = 57; // temp user id for development to bypass auth
+
+  useEffect(() => {
+    const fetchFriendsList = async () => {
+      const friends = await getFriendsList(userId);
+
+      const friendListPromise = friends.map(async (friend) => {
+        const friendInfo = await getUserById(friend.friend_user_id);
+        return friendInfo;
+      });
+
+      const friendList = await Promise.all(friendListPromise);
```

Comment:
No need to do promise.all here since there's only one. You can just chain it instead of a new variable